cmd/earthly/root_cmds.go

```diff
@@ -754,3 +805,31 @@ func (app *earthlyApp) actionPrune(cliCtx *cli.Context) error {
 	app.console.Printf("Freed %s\n", humanize.Bytes(total))
 	return nil
 }
+
+func handleDockerFiles(buildContextPath string) error {
+	earthfilePath := filepath.Join(buildContextPath, "Earthfile")
+	earthfilePathExists, err := fileutil.FileExists(earthfilePath)
+	if err != nil {
+		return errors.Wrapf(err, "failed to check if %q exists", earthfilePath)
+	}
+	if earthfilePathExists {
+		return errors.Errorf("earthfile already exists; please delete it if you wish to continue")
+	}
+
+	dockerIgnorePath := filepath.Join(buildContextPath, ".dockerignore")
+	dockerIgnorePathExists, err := fileutil.FileExists(dockerIgnorePath)
+	if err != nil {
+		return errors.Wrapf(err, "failed to check if %q exists", dockerIgnorePath)
+	}
+
+	if !dockerIgnorePathExists {
+		return nil
+	}
+
+	earthlyIgnoreFilePath := filepath.Join(buildContextPath, ".earthlyignore")
+	err = copy.Copy(dockerIgnorePath, earthlyIgnoreFilePath)
```

Comment:
this feels odd -- as I user, I would not expect running `earthly docker-build ...` to produce any files in my current working directory.

Reply:
Is this still being produced in the CWD rather than the temp dir?

Could we somehow create this in the temp dir, and override https://github.com/earthly/earthly/blob/main/buildcontext/local.go#L94 for this particular case?

Reply:
removed, the command does not longer generate a `.earthlyignore` file